docs/user-guide/configure-overview.md

```diff
@@ -12,192 +12,161 @@ using environment variables with the same name as the property.
 ### Example styx-config
 
 ```yaml
-    # A string uniquely identifying the host running the application, must be different for all running instances of the application
-    # the default value is suitable only for non clustered environments
-    jvmRouteName: "${jvm.route:noJvmRouteSet}"
-
-
-    proxy:
-      connectors:
-      - type: http
-        # Server port for Styx proxy.
-        port: 8080
-
-      #  0 -> availableProcessors / 2 threads will be used
-      bossThreadsCount: 1
-
-      # styx client worker threads are those performing all the asynchronous I/O operation to the backend origins.
-      # 0 -> availableProcessors / 2 threads will be used
-      clientWorkerThreadsCount: 0
-
-      # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
-      # 0 -> availableProcessors / 2 threads will be used
-
-      workerThreadsCount: 0
-
-      tcpNoDelay: true
-      nioReuseAddress: true
-      nioKeepAlive: true
-      maxInitialLength: 4096
-      maxHeaderSize: 8192
-      maxChunkSize: 8192
-      maxContentLength: 65536
-
-
-      # A timeout for idle persistent connections, in milliseconds.
-      keepAliveTimeoutMillis: 12000
-
-      # Time in milliseconds Styx Proxy Service waits for an incoming request from client
-      # before replying with 408 Request Timeout.
-      requestTimeoutMillis: 12000
-
-
-    admin:
-      connectors:
-      - type: http
-        # Server port for Styx admin interface.
-        port: 9000
-
-      # Number of threads for handling incoming connections on admin interface. 0 -> availableProcessors / 2 threads will be used.
-      bossThreadsCount: 1
-
-      # Number of worker threads for admin interface
-      # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
-      # 0 -> availableProcessors / 2 threads will be used
-      workerThreadsCount: 1
-
-      tcpNoDelay: true
-      nioReuseAddress: true
-      nioKeepAlive: true
-      maxInitialLength: 4096
-      maxHeaderSize: 8192
-      maxChunkSize: 8192
-      maxContentLength: 65536
-      
-      # Whether to cache the generated JSON for the /admin/metrics and /admin/jvm pages
-      metricsCache:
-        enabled: true
-        expirationMillis: 10000
-
-    loadBalancing:
-      strategy: #Check load balancing documentation for all the possible strategies
-        factory: {class: "com.hotels.styx.client.loadbalancing.strategies.PowerOfTwoStrategy$Factory"}
- 
-
-
-      adaptive:
```

Comment:
I see some config examples that still use the adaptive strategy (like styx-config-test.yml or styx-config-alternative.yml).  Can you please replace those references as well?